pull_request_template.md

```diff
@@ -1,25 +1,36 @@
 <!-- First of all, THANK YOU for your contribution. -->
 
 ## Changes Proposed:
--  
--  
+-  [ ] This pull request promotes changes to the core (units, players, creatures, game systems).
+-  [ ] This pull request promotes changes to scripts (bosses, spell scripts, creature scripts).
+-  [ ] This pull request promotes database changes.
```

Comment:
I do think this is pretty redundant as the labels applied to the PR when it is opened tell you this. But I also get the reasoning for this

Reply:
But also, probably just redundant

Reply:
So my vote would be to remove it

Reply:
I believe for the most part the proposed changes is redundant as that should be contained in the title/commits in a concise way but at the same time I feel like the goal here is to expand and have an option to expand if necessary

Reply:
I agree with you partially. Because if you refactor something larger you might want to specify smaller things that were changed.

Anywho, these 3 points I do think serve no point as the labels show exactly this information.